src/src/TrackingContext.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect, useContext, useMemo } from "react";
+
+const TrackingContext = React.createContext(null);
+
+function TrackingProvider({ children }) {
+  let isEnabled = false;
+  if (window.env === "production") {
+    isEnabled = true;
+  }
+
+  useEffect(() => {
+    if (isEnabled) {
+      console.log("Tracking enabled");
+      var _mtm = window._mtm = window._mtm || [];
```

Comment:
Why not simply do `window._mtm = window._mtm || []` and use `window._mtm` for the next `push`-operation.
This is the only use of it here, and would seem more straight forward?

I might be missing something javascript-y, so please; Enlighten me senpai